trick_source/codegen/Interface_Code_Gen/main.cpp

```diff
@@ -160,7 +160,9 @@ int main(int argc, char * argv[]) {
     // Set all of the defaults to c++
 #if (LIBCLANG_MAJOR > 3) || ((LIBCLANG_MAJOR == 3) && (LIBCLANG_MINOR >= 9))
     llvm::Triple trip (to.Triple) ;
-#if (LIBCLANG_MAJOR >= 12)
+#if (LIBCLANG_MAJOR >= 15)
+    //clang::CompilerInvocation::setLangDefaults(ci.getLangOpts(), clang::Language::CXX, trip, ppo.Includes) ;
```

Comment:
This commented out code appears to be the only thing with the if (LIBCLANG_MAJOR >= 15) block. Can the comment be removed?